src/domain/collaboration/callout/callout.resolver.subscriptions.ts

```diff
@@ -92,4 +94,29 @@ export class CalloutResolverSubscriptions {
       SubscriptionType.CALLOUT_ASPECT_CREATED
     );
   }
+
+  @UseGuards(GraphqlGuard)
+  @TypedSubscription<CalloutMessageReceived, CalloutMessageReceivedArgs>(
+    () => CalloutMessageReceived,
+    {
+      description: 'Receive comments on Callouts',
+      filter(this: CalloutResolverSubscriptions) {
+        return true; // todo
+      },
+      resolve(this: CalloutResolverSubscriptions, payload) {
+        return payload; // todo
+      },
+    }
+  )
+  async calloutMessageReceived(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    @CurrentUser() agentInfo: AgentInfo,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    @Args({ nullable: false }) args: CalloutMessageReceivedArgs
+  ) {
+    // todo
```

Comment:
todo?

Reply:
The whole body of the initial subscription is missing - the authorization, and the logging.
Also, the filter and resolve methods are not implemented.